src/Automation/Automation/Common/AutomationPSClient.cs

```diff
@@ -448,6 +450,129 @@ public void DeleteModule(string resourceGroupName, string automationAccountName,
 
         #endregion
 
+        #region PowerShell72Module
+
+        public Module CreatePowerShell72Module(string resourceGroupName, string automationAccountName, Uri contentLink,
+            string moduleName)
+        {
+            var createdModule = this.automationManagementClient.PowerShell72Module.CreateOrUpdate(resourceGroupName,
+                automationAccountName,
+                moduleName,
+                new AutomationManagement.Models.ModuleCreateOrUpdateParameters()
+                {
+                    Name = moduleName,
+                    ContentLink = new AutomationManagement.Models.ContentLink()
+                    {
+                        Uri = contentLink.ToString(),
+                        ContentHash = null,
+                        Version = null
+                    },
+                });
+
+            return this.GetPowerShell72Module(resourceGroupName, automationAccountName, moduleName);
+        }
+
+        public Module GetPowerShell72Module(string resourceGroupName, string automationAccountName, string name)
+        {
+            try
+            {
+                var module =
+                    this.automationManagementClient.Module.Get(resourceGroupName, automationAccountName, name);
+                return new Module(resourceGroupName, automationAccountName, module);
+            }
+            catch (ErrorResponseException cloudException)
+            {
+                if (cloudException.Response.StatusCode == System.Net.HttpStatusCode.NotFound)
+                {
+                    throw new ResourceNotFoundException(typeof(Module),
+                        string.Format(CultureInfo.CurrentCulture, Resources.ModuleNotFound, name));
+                }
+
+                throw;
+            }
+        }
```

Comment:
I didn't see any difference between this function and L352-L370. Could you explain why a new set of PowerShell72 is needed?

Reply:
I have Updated the function. Thanks for pointing it